### DISCUSSION TITLE
The right column is missing. How to configure the display right column?

### DISCUSSION THREAD

**Discussion by yuminchao** [TriliumNext/Trilium#6237] [Q&A] [answered]
### Describe feature

Hello, the author, after I upgraded the latest version of the client, I found that the right column (the original calendar and notes in the right column) is missing. How to configure the display of the right column?

**Comment by zadam** [comment, marked as answer]
Hello, I guess you upgrade froma very old version? If so, there's no way to put calendar to the right panel. As you probably already noticed, it's here and it's not possible to move:

![](https://github.com/zadam/trilium/assets/617641/207f7a2c-cfcf-4009-a8c1-0cab7a4e2240)
